package.json

```diff
@@ -26,6 +26,7 @@
     "babel-loader": "^6.2.0",
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
-    "webpack-dev-server": "^1.14.0"
+    "webpack-dev-server": "^1.14.0",
+    "jquery": "^2.2.4"
```

Comment:
will remove jquery; 
originally added the semantic ui js file, which needed jquery, but then removed it;